core/services/vrf/listener_v2.go

```diff
@@ -651,6 +707,11 @@ func (lsn *listenerV2) processRequestsPerSub(
 				"attempts", p.req.attempts)
 
 			if p.err != nil {
+				if startBalanceNoReserveLink.Cmp(p.juelsNeeded) == -1 {
```

Comment:
is == -1 the same as < 0 here? We should probably standardize on one if they're the same

Reply:
Switched to < 0 since that's what we're using elsewhere.